internal/pkg/instrumentation/probe/sampling/sampling_linux_test.go

```diff
@@ -0,0 +1,130 @@
+//go:build ebpf_test
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Similar to the other files with build directives, we want to keep the copyright at the top.

```suggestion
// Copyright The OpenTelemetry Authors
// SPDX-License-Identifier: Apache-2.0

//go:build ebpf_test
```